core/agent_core.go

```diff
@@ -97,7 +106,8 @@ func (agent *Agent) Start() error {
 		return nil
 	case <-time.After(agent.MaxStartupTime):
 		//TODO FIX - stop the initialization and close already initialized
-		return fmt.Errorf("%s", "Some plugins not intialized before timeout")
+		return fmt.Errorf("plugin %v not completed before timeout. Initialized %d plugins of %d: %v",
+			agent.handledPlugin, len(agent.loadedPlugins), len(agent.plugins), agent.loadedPlugins)
```

Comment:
Just log using multiple lines the ok ones and the problematic ones

Reply:
sample log:
ERRO[0015] Error loading core: plugin DatasyncExample Init() not completed before timeout.
Init succ: [ServiceLabel StatusCheck ResyncOrch ETCD ETCDDataSync] 
Init err: [DatasyncExample] 
AfterInit succ: [] 
AfterInit err: [ServiceLabel StatusCheck ResyncOrch ETCD ETCDDataSync DatasyncExample]  loc="core/event_loop.go(27)" tag=00000000